code/modules/spells/spell_types/ethereal_jaunt.dm

```diff
@@ -12,7 +12,7 @@
 	include_user = 1
 	centcom_cancast = 0 //Prevent people from getting to centcom
 	nonabstract_req = 1
-	var/jaunt_duration = 50 //in deciseconds
+	var/duration = 50 //in deciseconds

```

Comment:
Mutation need a duration defined, figured I'd just consolidate it since the mute has no reason to be shorter/longer than the jaunt.